### ISSUE TITLE
open a class definition just "def", use upper-case convention for constants

### ISSUE THREAD

**Issue by vic** [bakkdoor/fancy#26]
See
   http://gist.github.com/609889

**Comment by vic** [comment]
I'd prefer reserving "def" for methods. 

But we could still just use "class" for classes.

```
class Something {
      def method {
           bla bla
      }
 }
```

Like it more this way.

**Comment by vic** [comment]
Open fancy classes with 'class Name {}' Closed by 4ce5e35527f096fd412393836a97461147f6075f

**Comment by bakkdoor** [comment]
Yup, just using class is alot nicer. thanks =)
